lib/cli/run.js

```diff
@@ -270,6 +272,14 @@ exports.builder = yargs =>
         }
       });
 
+      if (argv.file) {
+        const nonExistentFiles = argv.file.filter(file => !fs.existsSync(file));
```

Comment:
This will fail. This will check only if the path exists or not. You should also check if that is a file
`fs.statSync(file).isFile()`

Reply:
fix thanks!
```

  const notFiles = argv.file.filter(file => !fs.statSync(file).isFile());
        if (notFiles.length > 0)
          throw createParameterIsNotAFile(
            `${nonExistentFiles} are not a files`
          );](url)
```